pkg/arrai/sysl.arrai

```diff
@@ -45,10 +53,23 @@ let normalize = \m
             appDescription: app.attrs?('description')?.s:{},
             appPatterns: patterns(app),
             appAnnotations: app.attrs?:{} >>> \k \v v.s?:(v.a.elt >> .s),
+            appEndpoints: app.endpoints?:{} => \(@:epName, @value:ep)
+            (
+                :epName,
+                epDescription: ep.attrs?('description')?.s:{},
+                epPatterns: patterns(ep),
+                epAnnotations: ep.attrs?:{} >>> \k \v v.s?:(v.a.elt >> .s),
+                epParams: ep.param?:[] => \(@:i, @item:param) 
+                    (
+                        name: param.name,
+                        type: param.type,
+                    ),
+                epProtocol: protocol(//rel.union({patterns(ep), patterns(app)})),
+            ),
```

Comment:
This shouldn't be here. If you need the details of the endpoints of an app, you should be able to do `apps <&> model.endpoints` to get a set of endpoints with the appropriate app data merged into each.